spec/controllers/rents_controller_spec.rb

```diff
@@ -0,0 +1,98 @@
+require 'rails_helper'
+
+describe RentsController do
+  let!(:starting_date) { Date.new 2017, 1, 1 }
+  let!(:ending_date) { Date.new 2018, 1, 1 }
+  let!(:book) { create :book }
+  let!(:book2) { create :book }
+  describe 'GET #index' do
```

Comment:
What is the subject of this describe? It's seems to me more clear to specify subject after describe
```rb
describe 'GET #index' do
  subject(:http_response) { get :index }
```